test/unit/test_transaction.cpp

```diff
@@ -258,12 +260,32 @@ TEST_F(test_transaction, trivial_types_dup_iterate_by_key)
         EXPECT_CALL(api, mdb_cursor_open(test_txn, test_dbi, _))
             .WillOnce(DoAll(SetArgPointee<2>(cursor), Return(MDB_SUCCESS)));
 
+        EXPECT_CALL(
```

Comment:
Could you also please extend test for `iterate` in the same way to verify that `mdb_cursor_get` is called correctly (with `MDB_FIRST` and "empty" `MDB_val` as a key).